OSPSuite.Presentation.Importer/Presenters/ColumnMappingPresenter.cs

```diff
@@ -38,11 +39,11 @@ IApplicationController applicationController
          {
             var format = _availableFormats.First(f => f.Name == formatName);
             SetDataFormat(format, _availableFormats);
-            OnFormatChanged?.Invoke(format);
+            OnFormatChanged(format);
          });
       }
 
-      public event FormatChangedHandler OnFormatChanged;
+      public event FormatChangedHandler OnFormatChanged = delegate { };
```

Comment:
Always initialize an event handler to empty delegate. that way you do not have to always check for null
Also I'd suggest to use the generic construct EventHandler<T> instead of defining your own EventHandler type. 
So instead of FormatChangedHandler, specify EventHandler<Format> or whatever eventargs you have